js/tiles.js

```diff
@@ -803,6 +805,7 @@ const door = {
 
 const forest = {
     both: OpenDoors,
+    levelCode: 2,
```

Comment:
There are tiles and not levels. So I think, if we mix that up as it may confuse people that tiles can not be used across levels.